python/hail/expr/functions.py

```diff
@@ -1872,6 +1872,39 @@ def is_complex(ref, alt):
     return _func("is_complex", tbool, ref, alt)
 
 
+@typecheck(ref=expr_str, alt=expr_str)
+def is_strand_ambiguous(ref, alt):
+    """Returns ``True`` if the alleles are strand ambiguous.
+
+    Strand ambiguous allele pairs are as follows:
+
+    - ("A", "T")
+    - ("T", "A")
+    - ("G", "C")
+    - ("C", "G")
+
+    Examples
+    --------
+    .. doctest::
+
+        >>> hl.eval_expr(hl.is_strand_ambiguous('A', 'T'))
+        True
+
+    Parameters
+    ----------
+    ref : :class:`.StringExpression`
+        Reference allele.
+    alt : :class:`.StringExpression`
+        Alternate allele.
+
+    Returns
+    -------
+    :class:`.BooleanExpression`
+    """
+    alleles = hl.capture({("A", "T"), ("T", "A"), ("G", "C"), ("C", "G")})
```

Comment:
prefer single quotes for python, unless using strings that contain single quotes in them